src/EnergyPlus/MixedAir.cc

```diff
@@ -4739,11 +4733,32 @@ void OAControllerProps::CalcOAEconomizer(EnergyPlusData &state,
             if (state.dataAirLoop->OutsideAirSys(state.dataAirLoop->AirLoopControlInfo(AirLoopNum).OASysNum).NumComponents == 1) {
                 // no need to simulate OA System if only a mixer is used in the OutsideAirSystem
 
-                Par(1) = this->MixNode;
-                Par(2) = this->RetNode;
-                Par(3) = this->InletNode;
-                Par(4) = this->MixMassFlow;
-                SolveRoot(state, Acc, MaxIte, SolFla, OASignal, MixedAirControlTempResidual, OutAirMinFrac, 1.0, Par);
+                auto f = [&state, this](Real64 const OASignal) {
+                    using Psychrometrics::PsyTdbFnHW;
+                    Real64 OAMassFlowRate = 0.0;     // outside air mass flow rate [kg/s]
```

Comment:
I collapsed this lambda down much smaller by just declaring the variables where used, and eliminated a few unnecessary variables.